src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -44,55 +44,56 @@ const CreateConceptForm = props => (
               onChange={props.handleChange}
             >
               <option value="N/A" />
-              {classes.map(option => <option key={option}>{option}</option>)}
+              {classes.map(option => (
+                <option key={option}>{option}</option>
+              ))}
             </select>
           </div>
         )}
         {props.concept && (
           <div className="form-group col-md-5">
             <label htmlFor="class">Class</label>
             <div>
-              { (props.concept.toString().trim() === 'symptom-finding') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control symptom-finding"
-                  required
-                  onChange={props.handleChange}
-                >
-                  <option key="Symptom-Finding">Symptom-Finding</option>
-                  <option key="Symptom">Symptom</option>
-                  <option key="Finding">Finding</option>
-
-                </select>
-              </React.Fragment>
-              }
-
-              { (props.concept.toString().trim() === 'set') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control set"
-                  required
-                  onChange={props.handleChange}
-                >
-                  <option key="LabSet">LabSet</option>
-                  <option key="ConvSet">ConvSet</option>
-                  <option key="MedSet">MedSet</option>
-
-                </select>
-              </React.Fragment>
-              }
+              {props.concept.toString().trim() === 'symptom-finding' && (
+                <React.Fragment>
+                  <select
+                    id="class"
+                    name="concept_class"
+                    value={props.state.concept_class}
+                    className="form-control symptom-finding"
+                    required
+                    onChange={props.handleChange}
+                  >
+                    <option key="Symptom-Finding">Symptom-Finding</option>
+                    <option key="Symptom">Symptom</option>
+                    <option key="Finding">Finding</option>
+                  </select>
+                </React.Fragment>
+              )}
 
-              {(props.concept.toString().trim() !== 'symptom-finding' && props.concept.toString().trim() !== 'set') && (
-              <span className="btn btn-sm btn-light normal-cursor col-12 text-capitalize pt-3">
-                {props.concept}
-              </span>
+              {props.concept.toString().trim() === 'set' && (
```

Comment:
Same goes for this.

Reply:
Same here fixing elslint warning.

Reply:
I reverted all the changes that are not part of this ticket.

Reply:
@dondrzzy  Concerning the above question. That is not part of the ticket that is why I have reverted the changes.